README.md

```diff
@@ -309,7 +309,8 @@ If things break or seem otherwise lackluster, **please** consult the
   Notcurses will not make use of bitmap protocols unless the terminal positively
   indicates support for them, even if <code>NCBLIT_PIXEL</code> has been
   requested. Likewise, sextants (<code>NCBLIT_3x2</code>) won't be used without
-  Unicode 13 support, etc. <code>ncvisual_blit()</code> will use the best blitter
+  Unicode 13 support, octants (<code>NCBLIT_4x2</code>) won't be used without
+  Unicode 17 support, etc. <code>ncvisual_blit()</code> will use the best blitter
```

Comment:
have we traveled to the future in a Delorean? unicode 17 does not yet exist afaik